downloader/src/main/kotlin/VcsHost.kt

```diff
@@ -243,9 +243,7 @@ enum class VcsHost(
          */
         fun toVcsInfo(projectUrl: String): VcsInfo {
             val vcs = try {
-                URI(projectUrl).let {
-                    values().find { host -> host.isApplicable(it) }?.toVcsInfoInternal(it)
-                }
+                toVcsHost(projectUrl)?.toVcsInfoInternal(URI(projectUrl))
```

Comment:
This now converts `projectUrl` twice to a URI, once inside `toVcsHost()` and once as the argument to `toVcsInfoInternal()`, which seems unnecessary.